### DISCUSSION TITLE
Date de rendu du projet 1

### DISCUSSION THREAD

**Discussion by JONATHAN-THIEBAUD** [heig-vd-dai-course/heig-vd-dai-course#423] [Q&A] [answered]
Bonjour, 

Quand devons nous rendre le travail pratique 1 ?

Merci d'avance et bonne soirée !

**Comment by ludelafo** [comment, marked as answer]
Bonjour,

Vous trouverez les dates de rendu dans les supports de cours à l'adresse suivante : <https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/07-practical-work-1/COURSE_MATERIAL.md#submission>.

Bonne journée,
Ludovic
